### ISSUE TITLE
Build problem

### ISSUE THREAD

**Issue by ghost** [tomlokhorst/swift-json-gen#19]
Hi Tom,

while your tool is working pretty well for one part of my application, there are problems in another part that I cannot solve.

I get a variety of errors, for instance: 
...+JsonGen.swift:21:31: Type 'Array<_>' has no member 'decodeJson'7/.../EventList+JsonGen.swift:21:31: Type 'Array<_>' has no member 'decodeJson'

We're talking here of a struct with one property:

```
struct EventList {
var list: [Event]

init() {
self.list = []
}
}
```

Both, EventList and Event are input to swift-json-gen. 

More errors appear for Int and NSDate, but these types should be covered by swift-json-gen, shouldn't they?

Do you have any clue what might be the problem here?

Thanks,
Hardy

**Comment by ghost** [comment]
oops, I just read #12 - I forgot to copy JsonGen.swift to the project. This reduced the amount of errors significantly, but not completely...

**Comment by ghost** [comment]
I believe it would be useful to mention that in the usage guide.

**Comment by ghost** [comment]
It should also be mentioned that structs must have one init(...) for all of their properties. This is not a problem when building without swift-json-gen, but leads to errors when using it, as it seems to expect such an initializer.

**Comment by tomlokhorst** [comment]
Good point about the Usage section of the readme. The JsonGen.swift file is mentioned in the Installation section, but I've now repeated that again in the Usage section.

The better solution would of course be if you didn't have to manually copy the file, as per https://github.com/tomlokhorst/swift-json-gen/issues/12.

I've also created a new issue about the initializer https://github.com/tomlokhorst/swift-json-gen/issues/20. JsonGen should just generate that if it doesn't exist already.

I hope this covers your issues. Thanks for using JsonGen!
